.github/workflows/unit_tests.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
         python-version: 3.9
     - name: Install Python packages
       run: |
-        pip install --upgrade pip six setuptools flake8 flake8-import-order mypy>=0.800 black types-six types-python-dateutil
+        pip install --upgrade pip six setuptools flake8 isort>=4.3.5 mypy>=0.800 black types-six types-python-dateutil
```

Comment:
```suggestion
        pip install --upgrade pip six setuptools flake8 isort[colors]>=4.3.5 mypy>=0.800 black types-six types-python-dateutil
```
This makes sure that colorama is installed too.

Reply:
we don't require colorama though...

Reply:
If you add `color_output = true` to `pyproject.toml` and add `[colors]` then it will color the diffs.

Reply:
It has no option to colorize output that goes to a tty, so we can't pass it through without more work.